files/build_templates/sonic_debian_extension.j2

```diff
@@ -245,6 +247,12 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install azure-s
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install watchdog
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install futures
 
+{% if install_kubernetes == "y" %}
+# copy kubelet service files
+sudo cp $IMAGE_CONFIGS/kubernetes/kubelet/10-kubeadm.conf $FILESYSTEM_ROOT/etc/systemd/system/kubelet.service.d/
+sudo LANG=C chroot $FILESYSTEM_ROOT systemctl disable kubelet.service
```

Comment:
>disable [](start = 46, length = 7)

Why disable?

Reply:
Good question. As it is allowed by build time flags,it should be enabled.

Reply:
Enabled; Start will happen at the time of join.

Reply:
I did see any iteration on this line. Can we remove this line?

---
In reply to: [404421527](https://github.com/Azure/sonic-buildimage/pull/4374#discussion_r404421527) [](ancestors = 404421527)

Reply:
We want it disabled until join.